src/main/io/beeper.c

```diff
@@ -338,11 +340,20 @@ void beeperUpdate(timeUs_t currentTimeUs)
         return;
     }
 
+    #ifdef USE_DSHOT
+    if (!ARMING_FLAG(ARMED) && currentBeeperEntry->mode == BEEPER_RX_SET) {  
```

Comment:
Using it for everything is a hard ask, since things like low voltage alarms make only sense if they work while armed, and you don't really want to waste motor update bandwidth on beeping in that situation.